### PR TITLE
Added: Objective-C support for Omise iOS SDK

### PR DESCRIPTION
**Pull request by pitiphong-p**
This PR is a patch on Omise iOS SDK to support client's Objective-C code. Consist of 

1. Explicitly declared `@objc` keyword on every public APIs.
2. Added Objective-C compatible counterpart methods/types of Objective-C non-compatible public APIs.

### CODE REVIEW

**Comment by chakrit** [comment]
:+1:

File: OmiseSDK/CardNumber.swift [hunk 1]
```diff
@@ -15,7 +15,11 @@ public final class CardNumber {
             .first
     }
     
-    public static func format(pan: String) -> String {
+    @objc(brandForPan:) public static func __brand(pan: String) -> Int {
```
Comment (pitiphong-p) [inline OmiseSDK/CardNumber.swift]: Anyone has a better idea than this? I have 2 ideas at first,
1. Return the raw value of the CardNumber instance 
2. Declare a new case on CardNumber represents the invalid card.
I choose no. 1 for now

Comment (chakrit) [inline OmiseSDK/CardNumber.swift]: I think `NSNotFound` is 🆗 for this since the method is actually "finding" a brand in a sense.

**Comment by igroomgrim** [comment]
👍

**Pull request merged**